code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -401,20 +401,29 @@
 	req_components = list(
 		/obj/item/stack/sheet/plasteel = 5)
 
+/obj/item/circuitboard/machine/HFR_fuel_input/syndicate
+	build_path = /obj/machinery/atmospherics/components/unary/hypertorus/fuel_input/syndicate
+
 /obj/item/circuitboard/machine/HFR_waste_output
 	name = "HFR Waste Output (Machine Board)"
 	greyscale_colors = CIRCUIT_COLOR_ENGINEERING
 	build_path = /obj/machinery/atmospherics/components/unary/hypertorus/waste_output
 	req_components = list(
 		/obj/item/stack/sheet/plasteel = 5)
 
+/obj/item/circuitboard/machine/HFR_waste_input/syndicate
```

Comment:
`HFR_waste_input` does not seem correct here

Reply:
i dont have vscode on the computer im currently on but i should be able to webedit it